rpcclient/wallet.go

```diff
@@ -821,6 +821,97 @@ func (c *Client) PurchaseTicket(fromAccount string,
 		numTickets, poolAddress, poolFees, expiry, ticketFee).Receive()
 }
 
+// CreateUnsignedTicketAsync returns an instance of a type that can be used to get the
+// result of the RPC at some future time by invoking the Receive function on the
+// returned instance.
+//
+// See CreateUnsignedTicket for more details.
+func (c *Client) CreateUnsignedTicketAsync(fromAccount string,
+	spendLimit dcrutil.Amount, minConf *int,
+	ticketAddress dcrutil.Address, numTickets *int,
+	poolAddress dcrutil.Address, poolFees *dcrutil.Amount, expiry *int,
+	ticketFee *dcrutil.Amount) FutureCreateUnsignedTicketResult {
+	// An empty string is used to keep the sendCmd
```

Comment:
The formatting here seems off.  It's supposed to be at 80 cols with a tw of 4.  Please fix.

Reply:
I am not sure what tw means, but I believe it is the tab size and it is fixed now.